src/cq.rs

```diff
@@ -110,6 +117,7 @@ impl CompletionQueueHandle {
 impl Drop for CompletionQueueHandle {
     fn drop(&mut self) {
         unsafe { grpc_sys::grpc_completion_queue_destroy(self.cq) }
+        // TOOD: free cq_shadow.
```

Comment:
I think it would be good to address this before merging so we don't have a memory leak.